lib/prompt.js

```diff
@@ -14,7 +14,11 @@ function promptMasked (options) {
     stdin.setEncoding('utf8')
     stderr.write(ansi.eraseLine)
     stderr.write(ansi.cursorLeft)
-    stderr.write(options.prompt)
+    if (cli.console.mocking()) {
```

Comment:
Sure! so to be clear- that's the only line I need? I don't need to check if it's mocking or not?